packages/ts-docs-gen/src/api-items/api-definition-base.ts

```diff
@@ -54,22 +54,9 @@ export abstract class ApiDefinitionBase<TKind extends Contracts.ApiBaseItemDto =
         return `<${members}>`;
     }
 
-    protected GetMembers(members: Contracts.ApiItemReference[]): ApiDefinitions[] {
-        return GeneratorHelpers
-            .GetApiItemReferences(this.ExtractedData, members)
-            .map<[ApiItemReference, Contracts.ApiItemDto]>(x => [x, this.ExtractedData.Registry[x.Id]])
-            .map(([reference, apiItem]) => GeneratorHelpers.SerializeApiDefinition(this.ExtractedData, apiItem, reference));
-    }
-
-    protected MembersToText(render: ReferenceRenderHandler, members: ApiDefinitions[], tab: number = 0): string[] {
-        return Helpers.Flatten(
-            members.map(x => x.ToText(render).map(y => `${GeneratorHelpers.Tab(tab)}${y}`))
-        );
-    }
-
     protected SerializedTypeToString(render: ReferenceRenderHandler, apiType: ApiTypes | undefined): string {
         if (apiType == null) {
-            // TODO: Add Log for missing type.
+            GeneratorHelpers.LogWithApiItemPosition(LogLevel.Warning, this.ApiItem, "Missing type.");
             return "???";

```

Comment:
Maybe we should change `???` to some explicit string for example `MISSING_TYPE` and describe it in documentation?

Reply:
It goes with an error. I will create an issue to include about this in documentation. Changed from `???` to `Contracts.TypeKeywords.Unknown`.